iOS/UI/Migration/MangaGridView.swift

```diff
@@ -14,29 +14,35 @@ struct MangaGridView: View {
     var coverUrl: URL?
 
     var body: some View {
-        LazyImage(url: coverUrl, resizingMode: .aspectFill)
+        LazyImage(url: coverUrl) { state in
+            if let image = state.image {
+                image
+                    .resizable()
+                    .aspectRatio(contentMode: .fill)
+            }
```

Comment:
probably good to have a fallback image here like we do in the UIKit things
```suggestion
            } else {
                Image("MangaPlaceholder")
            }
```